src/main/java/com/spark/masterdata/team/service/TeamServiceHelper.java

```diff
@@ -0,0 +1,23 @@
+package com.spark.masterdata.team.service;
```

Comment:
move this class to helper package
package com.spark.masterdata.team.service.helper;